File: benchmark/results/ts-monolith-opus-script/run-004/a-mcp/output/src/orders/fulfillment/value-objects/shipment-route.ts
import { Id } from '../../../shared/value-objects/id';
import { Timestamp } from '../../../shared/value-objects/timestamp';

export interface ShipmentRoute {
  id: Id;
  createdAt: Timestamp;
  updatedAt: Timestamp;
  name: string;
  description: string;
}

File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/shipping/value_objects/ShipmentRoute.java
package com.metaengine.demo.shipping.value_objects;

import java.time.Instant;

/** ShipmentRoute value object. */
public record ShipmentRoute(
    String id,
    Instant createdAt,
    Instant updatedAt,
    String name,
    String description
) {
}
